python/lsst/ts/weatherstation/config_schema.py

```diff
@@ -38,7 +38,6 @@
         type: string
         enum:
         - lsst
-        default: lsst
     allOf:
     # For each supported weather station controller add a new if/then case below.
     # Warning: set the default values for each case at the model level
```

Comment:
I'm afraid you need to get rid of the allOf and update the comment. What you want to do is for each supported weather station add a new set of configs below. I suggest the following (and updating your code accordingly):
```
    # For each type of weather station controller add new config below
    lsst_config:
        properties:
          host:
            type: string
          port:
            type: number
          buffer_size:
            type: number
          timeout:
            type: number
```
This is option (1) in the email that I sent earlier today.

Reply:
Also if you anticipate running more than one weather station CSC on the summit, please add the:
```
instances:
    sal_index:
```
like ESS, and as I mentioned in the email. The big deal about this new configuration system (not just a salobj quirk) is that the summit configuration for all instances of WeatherStation needs to be expressed in a file (or rather, a single pair of files): _init.yaml followed by _summit.yaml.

Reply:
Done